scripts/13-ocp-iam-users.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Use this script to grant access to IBM Cloud users to ROKS cluster
+# Update the IAM_USERS list below with the list of user email addresses
+# then run the script ./13-ocp-iam-users.sh
+
+set -euo pipefail
+
+TOOLKIT_GROUP=${TOOLKIT_GROUP:-ibm-toolkit-users}
+IAM_TOOLKIT_GROUP=${IAM_TOOLKIT_GROUP:-ibm-toolkit-iam-users}
```

Comment:
I think the intention of this group is to create a group of self-provisioners only.
maybe use `ibm-toolkit-self-provisioners` remove all references to IAM